### DISCUSSION TITLE
Land Cover - Attention! error

### DISCUSSION THREAD

**Discussion by Soossee** [UMEP-dev/UMEP#473] [Q&A] [answered]
hello everyone, I am trying to test UMEP's SOLWEIG analysis but it keeps giving me problems with the land cover raster. Although the extent is exactly the same of all the others raster and all values are reclassified as per the tutorial (paved, buildings, grass, bare soil and water and all in integer format) I cannot proceed with the analysis due to this error: **"The land cover grid includes integer values higher (or lower) than UMEP-formatted land cover grid (should be integer between 1 and 7). If other LC-classes should be included they also need to be included in landcoverclasses_2016a.txt'**.
Do you have any idea why this happens? Thank you for any help! I attach pictures to make it clearer
![image (2)](https://user-images.githubusercontent.com/117759087/200622613-f14ebbb5-fd99-42cf-9f27-aab12711856a.png)
![image (3)](https://user-images.githubusercontent.com/117759087/200622831-79dbdbef-34bd-44b2-a96b-f48cbe12110e.png)

**Comment by biglimp** [comment, marked as answer]
Could it be that you have some Nodata pixels somewhere? They will produce this window.

**Reply by Soossee** [reply]
Hi Biglimp, as you can see i have no 'no data', I used the Fill NoData cells command to make sure there were no problems about this topic. Do you have other ideas? Thank you so much
![image](https://user-images.githubusercontent.com/117759087/200852883-14721850-0315-46ff-b9bd-24f83bedbbff.png)

**Reply by biglimp** [reply]
Can you attach your raster for me to have a look at?

**Reply by Soossee** [reply]
Hi Biglimp, here the raster, it cover all the municipal territory of Milan. Is there something wrong? I thank you very much for your availability, I have tried many things but it still doesn't work and it's a pity because I would like to be able to exploit the potential of EMEP.!

 [Land Use Milano Raster.zip](https://github.com/UMEP-dev/UMEP/files/9989865/Land.Use.Milano.Raster.zip)

**Reply by biglimp** [reply]
It seems that you didnt fill the nodata values, that is why it doesnt work. Run FillNoDataValues again and set Nodata to 1. See the result from my Map Canvas below.
![image](https://user-images.githubusercontent.com/11908356/201345467-914855a5-1ab3-4965-ac9b-d83ce33a393f.png)

**Reply by Soossee** [reply]
yes finally the analysis has started, I hope it won't give me any more problems! thank you very much

**Reply by Soossee** [reply]
hello, I sang victory too early... the analysis has indeed started but the outputs are not the same as in the tutorials: there are only 6 rasters as can be seen in the picture and not 24 as it should be, is this normal? I am attaching pictures to explain better, I hope you can help me this time too!
![image](https://user-images.githubusercontent.com/117759087/202442301-a5923287-3050-4135-aed7-39800cfc2032.png)
![image](https://user-images.githubusercontent.com/117759087/202442353-0f008367-090f-44f3-87c1-b3957d89f0ea.png)

**Reply by biglimp** [reply]
Can I have a look at your meteorological file?

**Reply by Soossee** [reply]
Sure here it is! thank you

[prova_meteo_milano_15ago.txt](https://github.com/UMEP-dev/UMEP/files/10031972/prova_meteo_milano_15ago.txt)

**Reply by biglimp** [reply]
Looks ok. I think it might be because of the size of your data:

Width
3707
Height
3314

This will take a looong time to calculate, especially during daytime hours. THat is maybe why you dont have any more result. I would tile my data and run them separately in order to reduce computer memory issues
